lib/sprinkler_web/live/dashboard_live.ex

```diff
@@ -0,0 +1,30 @@
+defmodule SprinklerWeb.DashboardLive do
+  use SprinklerWeb, :live_view
+
+  @telemetry_topic "telemetry"
+
+  @impl true
+  def mount(_params, _session, socket) do
+    Phoenix.PubSub.subscribe(Sprinkler.PubSub, @telemetry_topic)
+    {:ok, assign(socket, :devices, [%{id: "1", tmps: [2]}, %{id: "2", tmps: [2]}])}
```

Comment:
I'm not sure about how this structure should be initialized? What would be the default values?